wmn-data.json

```diff
@@ -4388,6 +4388,16 @@
         "known" : ["dev", "pydude"],
         "cat" : "coding"
        },
+       {
+        "name" : "npm",
+        "uri_check" : "https://www.npmjs.com/~{account}",
+        "e_code" : 200,
+        "e_string" : "/npm-avatar/",
+        "m_string" : "poster depicting npm's mascot wombat",

```

Comment:
Please check this in the HTML source as I don't see this string. I see this string with HTML entities `poster depicting npm&#x27;s mascot wombat`

Reply:
true